Add Best Practices and Example Apps Links (#528)
smartdevicelink/sdl_app_library_guides: docs/Getting Started/Integration Basics - JS/index.md

        </script>
    </body>
</html>
```

## Where to Go From Here

> You should now be able to connect to a head unit or emulator. From here, [learn about designing your main interface](Displaying a User Interface/Main Screen Templates). For further details on connecting, see [Connecting to a SDL Core](Getting Started/Connecting to an Infotainment System). To learn optimal design patterns for SDL checkout the [Best Practices guide](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/).

Reviewer: ```suggestion
You should now be able to connect to a head unit or emulator. From here, [learn about designing your main interface](Displaying a User Interface/Main Screen Templates). For further details on connecting to a test module or emulator, see [Connecting to a SDL Core](Getting Started/Connecting to an Infotainment System). To learn about best practices for an SDL app, see our [Best Practices guide](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/).
```

> You should now be able to connect to a head unit or emulator. From here, [learn about designing your main interface](Displaying a User Interface/Main Screen Templates). For further details on connecting to a test module or emulator, see [Connecting to a SDL Core](Getting Started/Connecting to an Infotainment System). To learn about best practices for an SDL app, see our [Best Practices guide](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/).

Reviewer: ```suggestion
You should now be able to connect to a head unit or emulator. For more guidance on connecting, see [Connecting to an Infotainment System](Getting Started/Connecting to an Infotainment System). To start building your app, [learn about designing your interface](Displaying a User Interface/Main Screen Templates). Please also review the [best practices](https://smartdevicelink.com/en/guides/sdl-overview-guides/best-practices/display-information/) for building an SDL app.
```

This should be applied to the others as well.
